pkg/network/setup/netstat.go

```diff
@@ -21,11 +21,13 @@ package network
 
 import (
 	"fmt"
+	"net"
 	"slices"
 	"strings"
 	"sync"
 
 	"k8s.io/apimachinery/pkg/types"
+	netutils "k8s.io/utils/net"
```

Comment:
nit: why use named import ? there isn't a /net conflict in this file

optional FP (follow-up): script that checks if new PRs (not current code), has those - wdyt?

Reply:
`net` is a standard library of Go, knowing that makes it inappropriate (IMO) to use that name for a 3rd party package. If we'll need to use the std one, it will make less sense to alias it and in the end, it make make the code confusing (especially in the network domain) to see `net` ref something else.